OsmAnd/src/net/osmand/plus/views/layers/MapTileLayer.java

```diff
@@ -74,21 +89,37 @@ public void initLayer(@NonNull OsmandMapTileView view) {
 
 		useSampling = Build.VERSION.SDK_INT < 28;
 
-		paintBitmap = new Paint();
-		paintBitmap.setFilterBitmap(true);
-		paintBitmap.setAlpha(getAlpha());
-		
+		if (view.getMapRenderer() == null || paintBitmap == null) {
```

Comment:
&& ?

Reply:
view.getMapRenderer() == null ?? not needed?

Reply:
is it needed at all? probably we can always create paintBitmap and don't pay attention to it cause user can always switch between opengl & non-opengl version. It will be for couple releases anyway

Reply:
Will remove condition.